src/mon/OSDMonitor.cc

```diff
@@ -3547,6 +3547,34 @@ void OSDMonitor::get_health(list<pair<health_status_t,string> >& summary,
     }
 
     if (osdmap.test_flag(CEPH_OSDMAP_REQUIRE_LUMINOUS)) {
+      // An osd could configure failsafe ratio, to something different
+      // but for now assume it is the same here.
+      float fsr = g_conf->osd_failsafe_full_ratio;
+      if (fsr > 1.0) fsr /= 100;
+      float fr = osdmap.get_full_ratio();
+      float br = osdmap.get_backfillfull_ratio();
+      float nr = osdmap.get_nearfull_ratio();
+
+      // These checks correspond to how OSDService::check_full_status() in an OSD
+      // handles the improper setting of these values.
+      if (br < nr) {
+	ostringstream ss;
+	ss << "Backfill ratio (" << br << ") < nearfull ratio (" << nr << "), increased";
```

Comment:
can we make these errors use teh literal name?  "full_ratio", "backfillfull_ratio", etc.?